src/transformers/generation/configuration_utils.py

```diff
@@ -285,6 +288,16 @@ class GenerationConfig(PushToHubMixin):
         max_matching_ngram_size (`int`, *optional*, default to `None`):
             The maximum ngram size to be considered for matching in the prompt. Default to 2 if not provided.
 
+        > Generation parameters exclusive to [DoLa decoding](https://arxiv.org/abs/2309.03883)
+
+        dola_layers (`str` or `List[int]`, *optional*):
+            The layers to use for DoLa decoding. If `None`, DoLa decoding is not used. If a string, it must
+            be one of "low" or "high", which means using the lower part or higher part of the model layers, respectively,
```

Comment:
This isn't clear - what does it mean to use the "higher" parts of a layer?

Reply:
I have added one more sentence to explain `low` and `high` here! Let me know if it's clear enough!